runner/runners/combine.go

```diff
@@ -0,0 +1,44 @@
+package runners
```

Comment:
pattern I've used is to have
- X'ers - providing entry points to X
- X'erHelper - providing utility routines for implementing X (hidden to all users of X)
- X'erManager - typically provides X entry points, plus additional API, as necessary, for handling request volume and multiple Xers and using inner X'ers to actually do the work
So we would have: 
RunnerAndStatus
	QueuingRunner/RunnerManager (I like either name)
		Runner
			RunnerHelper
				Execer
					OsExecer
	Status
I've put a sample architecture diagram and component list on https://docs.google.com/document/d/1fKU6Pp8kmXKYuzV-pH47ODJwTn3Isdp64KY32AQammA/edit# (this content would be put in the readme.md)